bedrock/firefox/templates/firefox/desktop/trust.html

```diff
@@ -48,25 +48,28 @@
       </p>
 
       <ul class="feature-list">
-        <li id="privacy-dnt">
-          <h3>{{ _('Do Not Track') }}</h3>
+        {% if l10n_has_tag('tracking_protection') %}
```

Comment:
We probably want to add a waffle switch here, so the new copy only shows if the locale has the l10n flag **and** the waffle switch is `true`, e.g. something like:

    {% if l10n_has_tag('tracking_protection') and waffle.switch('tracking_protection') %}